packages/next/src/build/webpack-config.ts

```diff
@@ -389,14 +389,15 @@ export function getDefineEnv({
   }
 }
 
-function createReactAliases(
+function createRSCAliases(
```

Comment:
Renamed as it no longer contains React specific aliases, but also `server-only` and `client-only`.